Scripts/Audio/Audio.cs

```diff
@@ -27,7 +27,7 @@ public static void GetAudioClip(string encoding, byte[] bytes, Action<AudioClip>
                 throw new NullReferenceException("There was no audio data to generate from. Check your audio settings.");
             }
 
-            MainThreadConsumer.Instance.Consume(GenerateAudio(encoding, bytes, onAudioGenerated));
+            MainThreadDispatcher.Instance.Consume(GenerateAudio(encoding, bytes, onAudioGenerated));
```

Comment:
The MainThreadDispatcher re-structuring change is really good, makes sense to me.